snp_motion_planning/src/planner_profiles.hpp

```diff
@@ -133,8 +134,27 @@ tesseract_planning::OMPLDefaultPlanProfile::Ptr createOMPLProfile(
 std::shared_ptr<tesseract_planning::TrajOptPlanProfile> createTrajOptToolZFreePlanProfile()
 {
   auto profile = std::make_shared<tesseract_planning::TrajOptDefaultPlanProfile>();
-  profile->cartesian_coeff = Eigen::VectorXd::Constant(6, 1, 5.0);
-  profile->cartesian_coeff(5) = 0.0;
+  Eigen::VectorXd cart_coeff = Eigen::VectorXd::Constant(6, 1, 2.5);
+  cart_coeff(5) = 0.0;  // Set yaw to no cost
+  Eigen::VectorXd cart_tolerance = Eigen::VectorXd::Zero(6);
+  cart_tolerance << 0.01, 0.01, 0.01, 0.05, 0.05, 0.05;  // x, y, z, roll, pitch, yaw
```

Comment:
I don't think we can have tool z-axis free unless the tolerance on yaw is high

```suggestion
  cart_tolerance << 0.01, 0.01, 0.01, 0.05, 0.05, std::numeric_limits<double>::inf();  // x, y, z, roll, pitch, yaw
```

Reply:
If you set the coefficient to zero then the cost is always zero so the tolerance doesn't actually matter

Reply:
Changing to be 6.28 radians since it's being set from a ROS param in a launch file now.